cli.js

```diff
@@ -32,14 +33,26 @@ const download = (url, cb) => {
   }).on('error', err => cb(err))
 }
 
+// Parse command line arguments
+const argumentDefinitions = [
+  {
+    name: 'outputPath',
+    type: String,
+  },
+]
+const args = commandLineArgs(argumentDefinitions)
+const outputPath = args.outputPath || './coverage/badge.svg'
```

Comment:
Looks like you could use commandLineArgs's [defaultValue](https://github.com/75lb/command-line-args/blob/9c07457557351d3db825bd5a3d17c053d6a10dea/doc/option-definition.md#module_option-definition--OptionDefinition+defaultValue) instead.

Reply:
thanks @tlvince When you get a chance can you please review the changes?